samples/README.MD

```diff
@@ -0,0 +1,44 @@
+# Samples, Snippets, and How-To Guides
+
+Developers like to learn by looking at code, and so the Azure SDK comes with a myriad of code samples in the form of short code snippets, sample applications, and how-to guides. This document describes where to find all these resources.
+
```

Comment:
I would here quickly list them (and link to corresponding sections below):
- READMEs
- Samples per service
- docs.microsoft.com/samples
- docs.microsoft.com Service pages with QS and Tutorials
- reference documentation